index.md

```diff
@@ -83,37 +62,58 @@ We weight grades as follows:
+- [80, 90): B-, B, B+
+- [70, 80): C-, C, C+
+- [60, 70): D-, D, D+
+- [0, 60): F
 
+## Assignments ##
+We expect to have 4 assignments over the course of the semester. In each of these, students will work in pairs to solve a specific implementation problem in the virtual reality lab in 4107 Siebel Center.
 
-## Textbooks ##
-Main text, required: [Steven M. LaValle, Virtual Reality, 2016](http://vr.cs.uiuc.edu).
+- Read the submission instructions in each handout carefully!
+- You must work in pairs on all assignments. We recommend using Piazza to find a group.
+- We will accept assignments only up to two days late, with a 20% penalty.
 
-Optional: George Mather, Foundations of Sensation and Perception: Psychology Press; 2 edition, 2009.
+See the [Assignments page](/assignments) for more details.
 
-Optional: Peter Shirley, Michael Ashikhmin, and Steve Marschner, Fundamentals of Computer Graphics, A K Peters/CRC Press; 3 edition, 2009.
+## Exams ##
+This class will have three midterms and no final exam. Each exam is 50 minutes long. You will need to make a reservation in the [CBTF](https://cbtf.engr.illinois.edu) to take the exam.
 
-See the [Extra Material](/extra-material) page for other, recommended readings.
+- Midterm 1: Feb. 10 - Feb. 12
+- Optional second chance exam for Midterm 1: Feb. 17 - Feb. 19
+- Midterm 2: Mar. 10 - Mar. 12
+- Optional second chance exam for Midterm 2: Mar. 25 Mar. 26
+- Midterm 3:  Apr. 28 - Apr. 30
 
-## [Piazza](https://piazza.com/illinois/spring2019/cs498vr) ##
-We will use Piazza for answering questions and administering the course.
+All exams are closed-everything (no book, notes, calculator, etc.). See the [CBTF policies](https://cbtf.engr.illinois.edu/for-students/policies.html) for more detail. You are responsible for all material covered in lectures and assignments.For DRES accommodations see [CBTF DRES Students](https://cbtf.engr.illinois.edu/for-students/dres-students.html)
 
-Click the header to join/view the forum. You do not need a passcode. If you want to use a non-`illinois.edu` email address on Piazza, then send an email from your `illinois.edu` mail account to the course instructor with the email address you want to use, and we will add you manually.
+## Second-chance Exams ##
+For the first two midterms, you have the option to take a second chance exam if you wish. **The maximum score you can achieve on the second chance exam is 88%.** Specifically, if your second chance score is N% your final percentage score will be calculated as 0.88 x 0.N x 100.0. Your second chance score will replace your first score, even if it is lower. The questions on the second chance exam will cover the same topics as the first exam but may not be identical to the questions on the first exam. To take the second chance exam, just make a reservation with CBTF. You do not need to inform course staff that you are doing so.
 
-## Course Staff ##
-See the [schedule for office hours](/officehours).
+## Regrade Requests ##
```

Comment:
I remember talking about moving assignment policies here, but now I'm not sure if they're better here than on the assignments page :/
But then it also makes sense to state the regrade policies for exams here because it definitely shouldn't be on the assignments page.
Would it be helpful to dedicate a page or section to all of the grading stuff? I think that way the students can find info about grading easier and bother us less. But there's not really space left for a new page

Reply:
I don't mind if all the course policies are on the same page, once we have a table of contents it'll make it much easier to find and until then, I don't think scanning the titles is that bad.